src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.ML
     public static class OnnxCatalog
     {
         /// <summary>
-        /// Applies a pre-trained Onnx model.
+        /// Create a <see cref="OnnxScoringEstimator"/>, which applies a pre-trained Onnx model to the input column.
         /// </summary>
         /// <remarks>
         /// All column names are provided, the input data column names/types must exactly match
```

Comment:
>All column names are provided, the input data column names/types [](start = 12, length = 64)

The name/type of input columns must exactly match name/type of the ONNX model inputs. The name/type of the produced output columns will match name/type of the ONNX model outputs.